src/commands/prompt/index.ts

```diff
@@ -105,7 +115,34 @@ async function runInternal(initialPrompt: string, options: PromptOptions) {
 
   output.outputVerbose(`Using model: ${config.model}`);
 
+  let fileContent = '';
+  if (options.file) {
+    const filePath = `${process.cwd()}/${options.file}`;
+
+    if (!fs.existsSync(filePath)) {
+      throw new Error(`Couln't find provided file: ${options.file}`);
+    }
+    fileContent = fs.readFileSync(filePath).toString();
+
+    const encoder = get_encoding('cl100k_base');
+    const tokens = encoder.encode(fileContent);
+
+    if (tokens.length <= 2000) {
+      output.outputInfo(`File you provided adds: ~${tokens.length} tokens to conversation`);
+    } else {
+      output.outputWarning(
+        `File you provided adds: ~${tokens.length} tokens to conversation. This might impact the cost.`
+      );
+    }
+  }
+
   const messages: Message[] = [];
+  const internalMessages: Message[] = [];
+
+  if (fileContent !== '') {
+    internalMessages.push({ role: 'system', content: DEFAULT_FILE_PROMPT });
+    internalMessages.push({ role: 'system', content: fileContent });
```

Comment:
Idk but shouldn't we do something like:
const filePrompt = ```
Answer based on the following content.

CONTENT:
${fileContent}
```

Reply:
Simplified along with system prompt